code/modules/urist/modules/wfc/modules/deepmaint/admins.dm

```diff
@@ -0,0 +1,36 @@
+// Adminbuse stuff for deepmaint
+
+/datum/admins/proc/generate_deepmaint()
+	set category = "Fun"
+	set desc = "(Re-)Generate DeepmaintRooms; place the map template first!"
+	set name = "Generate DeepmaintRooms"
+
+	if(!GLOB.wfc_deepmaint_zlevels_by_instance)
+		log_and_message_admins("- failed to generate Deepmaint - no Deepmaint Z-levels registered!")
```

Comment:
Super nit-pick. Leading Hyphen might look a little odd and out of place with the admin log